catalyst/dl/core/state.py

```diff
@@ -53,4 +52,49 @@ def __init__(
         )
 
 
-__all__ = ["State"]
+class GanState(_State):
```

Comment:
I don't think it's Core, it better to place at `catalyst/dl/state/gan.py`, just at SupervisedRunner

Reply:
Done